examples/batches/scan_form_batch/scan_form_batch.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+    "encoding/json"
+    "fmt"
+    "github.com/EasyPost/easypost-go"
+    "os"
+)
+
+func main() {
+    apiKey := os.Getenv("EASYPOST_API_KEY")
+    if apiKey == "" {
+        fmt.Fprintln(os.Stderr, "missing API key")
+        os.Exit(1)
+        return
+    }
+    client := easypost.New(apiKey)
+
+    // ScanForm Batch
+    batch, err := client.CreateBatchScanForms(
+        "batch_12345",
+        "string",
+    )
+    if err != nil {
+        fmt.Fprintln(os.Stderr, "error creating batch:", err)
```

Comment:
This should be `error creating scanform`

Reply:
how about `error creating ScanForm for Batch:` to avoid having the same error message as create_scanform.go line 25?

Reply:
Works for me!